projects/storefrontlib/cms-structure/routing/default-routing-config.ts

```diff
@@ -8,18 +8,20 @@ import { FeatureToggles, RoutingConfig } from '@spartacus/core';
 import { inject } from '@angular/core';
 
 export const defaultRoutesConfigFactory: () => RoutingConfig = () => {
+  const featureToggles = inject(FeatureToggles);
   const enableProductPageRouteAllowsNoProductName =
-    inject(FeatureToggles).defaultProductPageRouteAllowsNoProductName;
+    featureToggles.defaultProductPageRouteAllowsNoProductName;
+  const isNewAuthFlow = featureToggles.authorizationCodeFlowByDefault;
 
-  return {
+  const routingConfig = {
     routing: {
       routes: {
         home: { paths: [''] },
         notFound: { paths: ['not-found'] },
 
         // semantic links for login related pages
         login: {
-          paths: ['login'],
+          paths: [isNewAuthFlow ? 'signIn' : 'login'],
```

Comment:
`major:` for the consistency, let;s change the route name to lowercase
```suggestion
          paths: [isNewAuthFlow ? 'signin' : 'login'],
```

Apart from that, let's add a comment with description why this change is required. It is not obvious and might be difficult for other devs to understand.

Reply:
are you sure `signin`, not `sign-in`?